src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -674,9 +680,6 @@ def _prepare_inputs_with_constants(
                     new_input.quantizer.is_qat
                     and not input_.quantizer.is_precomputed_qat
                     and self.error_tracker is not None
```

Comment:
basically:
- only mixing ops and reshape ops (ops that quantize their inputs) call this function with `quantize_actual_values`
- only QuantizedArray that have `is_precomputed_qat` are now accepted for these functions if `is_qat` (QAT import was requested by the user).
- you don't neeed `QuantIdentity` before reshape/mixing layers applied on other mixing ops outputs because the "passthrough" for `is_precomputed_qat` implemented above.